deploy/demo/deployment.tpl

```diff
@@ -70,6 +70,11 @@ spec:
                 secretKeyRef:
                   name: rds-output
                   key: database_password
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: sentry-output
+                  key: sentry-dsn
```

Comment:
This code assumes Cloud Platform created the secret and Kubernetes will pass it through. 
We can add this to `secret.tpl` so that it can pick up the GitHub variable, like this:

```
apiVersion: v1
kind: Secret
metadata:
  name: justice-gov-uk-dev-secrets
type: Opaque
stringData:
  SENTRY_DSN: "${SENTRY_DSN}"
  GOV_NOTIFY_API_KEY: "${GOV_NOTIFY_API_KEY}"
  AUTH_KEY: "${AUTH_KEY}"
  ......
```